api/resolvers/item.js

```diff
@@ -398,6 +400,27 @@ export default {
             orderBy: orderByClause(by || 'zaprank', me, models, type)
           }, ...whenRange(when, from, to || decodedCursor.time), decodedCursor.offset, limit, ...subArr)
           break
+        case 'random':
+          items = await itemQueryWithMeta({
+            me,
+            models,
+            query: `
+              ${selectClause(type)}
+              ${relationClause(type)}
+              ${whereClause(
+                '"Item"."deletedAt" IS NULL',
+                '"Item"."weightedVotes" > 10',
```

Comment:
This probably shouldn't go here?

Also, 10 is just a random number as I don't know what it should be. It seems that no results are return if I set it to about 25.